AnkiDroid/src/main/java/com/ichi2/utils/LanguageUtil.kt

```diff
@@ -42,88 +42,88 @@ object LanguageUtil {
         "Afrikaans" to "af", // Afrikaans
         "አማርኛ" to "am", // Amharic
         "العربية" to "ar", // Arabic
-        "azərbaycan" to "az", // Azerbaijani
-        "беларуская" to "be", // Belarusian
-        "български" to "bg", // Bulgarian
+        "Azərbaycan" to "az", // Azerbaijani
+        "Беларуская" to "be", // Belarusian
+        "Български" to "bg", // Bulgarian
         "বাংলা" to "bn", // Bangla
-        "català" to "ca", // Catalan
+        "Català" to "ca", // Catalan
         "کوردیی ناوەندی" to "ckb", // Central Kurdish
-        "čeština" to "cs", // Czech
-        "dansk" to "da", // Danish
+        "Čeština" to "cs", // Czech
+        "Dansk" to "da", // Danish
         "Deutsch" to "de", // German
         "Ελληνικά" to "el", // Greek
         "English" to "en", // English
-        "esperanto" to "eo", // Esperanto
-        "español (Argentina)" to "es-AR", // Spanish (Argentina)
-        "español (España)" to "es-ES", // Spanish (Spain)
-        "eesti" to "et", // Estonian
-        "euskara" to "eu", // Basque
+        "Esperanto" to "eo", // Esperanto
+        "Español (Argentina)" to "es-AR", // Spanish (Argentina)
+        "Español (España)" to "es-ES", // Spanish (Spain)
+        "Eesti keel" to "et", // Estonian
+        "Euskara" to "eu", // Basque
         "فارسی" to "fa", // Persian
-        "suomi" to "fi", // Finnish
+        "Suomi" to "fi", // Finnish
         "Filipino" to "fil", // Filipino
-        "français" to "fr", // French
+        "Français" to "fr", // French
         "Frysk (Nederlân)" to "fy-NL", // Western Frisian (Netherlands)
         "Gaeilge (Éire)" to "ga-IE", // Irish (Ireland)
-        "galego" to "gl", // Galician
+        "Galego" to "gl", // Galician
         "Gothic" to "got", // Gothic
         "ગુજરાતી (ભારત)" to "gu-IN", // Gujarati (India)
         "עברית" to "heb", // Hebrew
         "हिन्दी" to "hi", // Hindi
-        "hrvatski" to "hr", // Croatian
-        "magyar" to "hu", // Hungarian
-        "հայերեն (Հայաստան)" to "hy-AM", // Armenian (Armenia)
+        "Hrvatski" to "hr", // Croatian
+        "Magyar nyelv" to "hu", // Hungarian
+        "Hայերեն (Հայաստան)" to "hy-AM", // Armenian (Armenia)
         "Indonesia" to "ind", // Indonesian
-        "íslenska" to "is", // Icelandic
-        "italiano" to "it", // Italian
+        "Islenska" to "is", // Icelandic
```

Comment:
```suggestion
        "íslenska" to "is", // Icelandic
```

> Language names as a rule are written in lowercase letters, likewise adjectives which are derived from nationalities, ex. íslenzka (Icelandic-language), enska (English language), danska (Danish language); íslenzkur (Icelandic person, adj.), enskur (English person, adj.), þýzkur (German person, adj.)

https://learn-nordic.livejournal.com/63751.html

----

Or use a capital í, instead of I